### DISCUSSION TITLE
lnd

### DISCUSSION THREAD

**Discussion by zhangtiantian222** [lightningnetwork/lnd#7282] [General]
I don't have a public IP, how can others connect to my node and complete the transfer? Create a two-way channel?

**Comment by ellemouton** [comment]
Hi @zhangtiantian222! 

If you just want to send and receive (not route), then not having a public IP is fine but you will need to be the one to open channels with some peers. These channels can be public or private.

If, however, you want other people to open channels to you so that you can route their payments & get free inbound liquidity then you will need to have a public IP _and_ will first need to open a _public_ channel yourself since your node will not be allowed to broadcast its IP address before you have a public channel

**Comment by zhangtiantian222** [comment]
I use the openchannel public node, but can only transfer out but not in, because the remote_balance is 0, can a two-way channel be established?

**Comment by ellemouton** [comment]
every channel is a two-way channel. The issue is that because you opened the channel (and I assume have not yet transferred much of it over to your peer) most of the funds are still on your side & not enough on your peers side & thus you dont have much inbound. You will either need to [Loop](https://docs.lightning.engineering/lightning-network-tools/loop) out some of your funds so that you have some inbound on that channel. Or you can use [Pool](https://docs.lightning.engineering/lightning-network-tools/pool) to buy inbound
